website/package.json

```diff
@@ -20,8 +20,8 @@
     "clsx": "1.1.1",
     "docusaurus-plugin-fathom": "1.1.0",
     "framer-motion": "4",
-    "react": "17.0.1",
-    "react-dom": "17.0.1"
+    "react": "^16.13.1",
```

Comment:
Would be better to keep using react 17.X, can you please revert?

Reply:
Building the `website` part with react 17.X gave me a lot of build warnings of other incompatible dependency packages,   
which I wasn't able to resolve quickly on my own.

I hope that updating the `website` part to react 17.X, can be done through a separate issue.

Reply:
Well the issue is that we were using react 17 before this pr, so I am wondering what changed that make it incompatible :)

Reply:
> Well the issue is that we were using react 17 before this pr, so I am wondering what changed that make it incompatible :)

I think it may still be compatible with react `v17.X`, however it will throw a lot of build warnings.   
Due to the dependency packages of the `website` component still being for react `v16.X`.

That's why downgrading to `v16.X` for the `website` component cleared a lot of my build warnings.   
And also why I believe it would be better off in a separate issue.

I've took the liberty of opening one here: https://github.com/ledokku/ledokku/issues/423

Reply:
I do get that the warnings are annoying but it's still compiling and the current version is working fine with 17.X but may have issues if we downgrade it. Wdyt about instead having an issue to fix the warnings in a separate pr?